test/AuthTokenVerifier.test.ts

```diff
@@ -45,56 +43,53 @@ describe("Testing AuthTokenVerifier", () => {
       sub: secondDID
     }
 
-    firstSigner = new JWT(firstKeys);
-    secondSigner = new JWT(secondKeys)
-    firstClaimToken = await firstSigner.sign(firstPayload, { algorithm: 'ES256' });
-    secondClaimToken = await secondSigner.sign(secondPayload, { algorithm: 'ES256' });
-  })
-
-  it("Checks if basic signing verification works", async () => {
-    const testSignature = await secondSigner.sign('Initial signed content', { algorithm: 'ES256' })
-    const secondPubKey = secondKeys.publicKey
-    const decoded = await secondSigner.verify(testSignature, secondPubKey)
-    
-    assert.strictEqual(decoded, 'Initial signed content')
+    firstClaim = await createSignWithEthersSigner(firstWallet)(firstPayload);
+    secondClaim = await createSignWithEthersSigner(secondWallet)(secondPayload);
   })
 
   it("should authenticate first DID with first Claim issuer", async () => {
```

Comment:
I know this isn't your change, but I think it would be better if the test names actually represented the use case that was being tested. More of a `why` the test is there, less of a `what`. So in this case, it would be
```suggestion
  it("should authenticate DID with created DID doc (owner pubKey) and ethers signature", async () => {
```
If you like this direction @JGiter @knzeng-e , we can try to change the other tests description.

Reply:
I see what you mean, I'm ok with your proposal.